ietf-scitt-charter.md

```diff
@@ -45,34 +45,34 @@ The main deliverables defined by this program of work provide a guideline for mi
 
 The WG shall start out by documenting and defining terms in an architectural model for:
 
-1. essential classes of actors, such as the supply chain "issuer" (one which generates supply chain artifacts and statements about them) and
+1. essential actors, such as the supply chain "issuer" (one which generates supply chain artifacts and statements about them) and
 2. the basic interactions these have with other actors, and their duties in the ecosystem.
 
 The architectural model shall provide an aggregated overview of corresponding actor-specific information models and interaction models and will provide examples of composition patterns that illustrate how to addresss a concise set of use cases.
 
-## Consistent Actor Identification
+## Establish consistent Identity Format for Actors
```

Comment:
I still think that is too strong of a statement, tbh. I would not drag the process or the format in here.

More examples of well-understood roles is fine.

Reply:
ok, let me think to soften a bit without losing the context! Thanks for the review!